includes/model/class-user.php

```diff
@@ -277,4 +280,50 @@ public function get_indexable() {
 			return false;
 		}
 	}
+
+	/**
+	 * Update User profile attributes
+	 *
+	 * @param string $key The attribute to update.
+	 * @param mixed $value The new value.
+	 *                     Possible values:
+	 *                   - name: The User-Name.
+	 *                   - summary: The User-Description.
+	 *                   - icon: The User-Icon.
+	 *                   - header: The User-Header-Image.
+	 * @return bool True if the attribute was updated, false otherwise.
+	 */
+	public function save( $key, $value ) {
```

Comment:
I would rename that, so that we will be able to have a generic `save` in the future, that saves all attributes to the DB.

Reply:
Happy to have dedicated functions. I went in an `update_key` direction with it.

Reply:
Sorry for maybe not being precise enough: My proposal was to simply not use the generic `save` function, but something different instead. I am fine with having one function with `key` => `value` params, but I am also fine with dedicated functions :)